src/main/java/fibonacci/Fibonacci.java

```diff
@@ -22,7 +29,17 @@ else if(number == 1){
         else{
             rval = fib(number - 1) + fib(number - 2);
         }
-
+        fibNumber = rval;
         return rval;
     }
+
+    @Override
+    public void run() {
+
+        fib(number);
```

Comment:
As mentioned above this returns the final number.  If you want a class level variable to hold the results, this is where it should happen not in a method that is called thousands of times.